go-controller/pkg/node/gateway_iptables.go

```diff
@@ -412,7 +412,7 @@ func getLocalGatewayNATRules(ifname string, cidr *net.IPNet) []iptRule {
 
 // initLocalGatewayNATRules sets up iptables rules for interfaces
 func initLocalGatewayNATRules(ifname string, cidr *net.IPNet) error {
-	return insertIptRules(getLocalGatewayNATRules(ifname, cidr))
+	return appendIptRules(getLocalGatewayNATRules(ifname, cidr))
```

Comment:
this part LGTM. can you please leave a comment above L415 inside the func on why we append ?

Reply:
since this isn't a bug upstream per se, would be good to have a line or two so that in future nobody changes this back to insert accidently.

Reply:
added